src/media/js/elements/header.js

```diff
@@ -26,10 +26,9 @@
         - Toggle all header children closed.
 */
 define('elements/header',
-    ['core/element_utils', 'core/z', 'document-register-element', 'jquery',
+    ['core/element_utils', 'core/z', 'document-register-element',
      'underscore'],
-    function(eUtils, z, dre, $,
-             _) {
+    function(eUtils, z, dre, _) {
```

Comment:
I think the `_` should stay where it was. These normally match the dependencies format,